rdrf/intframework/views.py

```diff
@@ -128,6 +128,13 @@ def _get_hub_response(self, registry_model, user_model, umrn: str) -> Optional[d
         logger.info(f"getting hub response for umrn {umrn}")
         hub = self._get_hub(registry_model, user_model)
         if hub is None:
+            result_model = HL7Message()
+            result_model.username = user_model.username
+            result_model.registry_code = registry_model.code
+            result_model.umrn = umrn
```

Comment:
do we allow the content to be empty - might be better to add result_model.content = " no message" if not